src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/__tests__/get_matched_indices.test.js

```diff
@@ -51,6 +51,52 @@ describe('getMatchedIndices', () => {
     });
   });
 
+  it('should support queries with wildcards in various places', () => {
+    const indices = [
+      { name: 'kibana' },
+      { name: 'es' },
+      { name: '.kibana' }
+    ];
+
+    const query = 'k*ib*';
+    const matchedIndices = [
+      { name: 'kibana' },
+    ];
+
+    const result = getMatchedIndices(indices, matchedIndices, query, false);
+
+    expect(result).toEqual({
+      allIndices: [{ name: 'kibana' }, { name: 'es' }],
+      exactMatchedIndices: [{ name: 'kibana' }],
+      partialMatchedIndices: [{ name: 'kibana' }],
+      visibleIndices: [{ name: 'kibana' }]
+    });
+  });
+
+  it('should support queries with wildcards in various places again', () => {
```

Comment:
What is this test verifying which the one before it does not?

Reply:
It should be verifying that a wildcard _not_ at the end of the query still works